testsuite/shared/pom.xml

```diff
@@ -165,6 +165,21 @@
             <artifactId>byteman</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.io7m.xom</groupId>
+            <artifactId>xom</artifactId>
+            <version>1.2.10</version>
```

Comment:
Use dependency-management in the root pom to specify versions, driven by a new version.xxxx property.